### DISCUSSION TITLE
Unable to login

### DISCUSSION THREAD

**Discussion by tismofied** [alexjustesen/speedtest-tracker#1665] [Q&A] [answered]
I migrated the container to a different host. migrated all config files and made necessary adjustments as far host ip and base key url.
the container starts and I get the login page but I can't get past that " These credentials do not match our records." I made sure I am putting the correct login info.

**Comment by alexjustesen** [comment, marked as answer]
You can use the `app:user-reset-password` command to reset your account's password.

Ref: https://docs.speedtest-tracker.dev/other/commands

**Reply by tismofied** [reply]
I don't know how I missed that. Thank you @alexjustesen
